synapse/federation/federation_client.py

```diff
@@ -238,13 +238,15 @@ async def claim_client_keys(
         self,
         destination: str,
         query: Dict[str, Dict[str, Dict[str, int]]],
+        requester: UserID,
```

Comment:
Can we use `user_id` or `user` and not `requester`? We almost exclusively use that to mean a `Requester` instance.